arwen/elrondapi/elrondei.go

```diff
@@ -22,6 +22,7 @@ package elrondapi
 // extern int32_t storageLoad(void *context, int32_t keyOffset, int32_t keyLength , int32_t dataOffset);
 // extern int32_t storageLoadFromAddress(void *context, int32_t addressOffset, int32_t keyOffset, int32_t keyLength , int32_t dataOffset);
 // extern void getCaller(void *context, int32_t resultOffset);
+// extern void nonPayableFuncCheck(void *context);
```

Comment:
If you have a better name in mind, it's not too late to change

Reply:
Maybe checkNonPayableFunc?

Reply:
Yeah ... thought about that, but I don't want it to seem like we are checking whether or not the function is non payable. We are checking that no payment came.

Reply:
I would say checkFunctionNonPayable

Reply:
Right, makes sense!